stdlib/Statistics.version

```diff
@@ -0,0 +1,2 @@
+STATISTICS_BRANCH = master
+STATISTICS_SHA1 = da6057baf849cbc803b952ef7adf979ae3a9f9d2
```

Comment:
Maybe slightly OT, but I noticed that it also seems possible to use the git tree SHA with the github API rather than the commit which I believe is also the way Pkg identifies the code for a version. f52ff30330bf9c2ddf91bc731c0cad78d9ec5b49 is the tree sha for this commit.

Reply:
It works (we tried it) , but the tarballs obtained from GitHub doesn't have a consistent hash.

Reply:
Hmm, odd. I just tried it and downloading https://api.github.com/repos/JuliaStats/Statistics.jl/tarball/f52ff30330bf9c2ddf91bc731c0cad78d9ec5b49 does appear to untar to a tree which is binary identical to Statistics.jl checked out at da6057b. I assume you refer to something else which is broken?

Reply:
I mean the hashsum of the tarball itself.

Reply:
So should I change something or not?

Reply:
No it looks like Pkg goes by the commit hash anyway.